sparse/usr/lib/systemd/system/set-low-memory-killer-values.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Set LMK killer thresholds and score groups
+After=init-done.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/bin/sh -c "echo 92160,115200,138240,161280,206490 > /sys/module/lowmemorykiller/parameters/minfree;echo 0,58,147,529,1000 > /sys/module/lowmemorykiller/parameters/adj"
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
We don't need this install section when we have symbolic link in place from target.wants

Reply:
:+1: removed